k-distribution/tests/regression-new/bison-parser-library/.gitignore

```diff
@@ -0,0 +1 @@
+test
```

Comment:
Why is this necessary?
I'm trying to avoid having files being created which are ignored by `git`, and also not cleaned by `make clean`

Reply:
My mistake - should be added to the make clean stanza for the custom test.